### DISCUSSION TITLE
Modal dialog will scroll to top on first opening, then it will scroll to the bottom

### DISCUSSION THREAD

**Discussion by barrytrendspark** [tailwindlabs/headlessui#2950] [Help]
I am using version `1.7.18`. I have a dialog modal that will scroll the entire content of my page to the top of the screen when first entering the dialog. On all subsequent dialog openings, the entire contents get scrolled down.

I have no idea why this is happening. Please can someone help.

**Comment by RejownAhmed** [comment]
Share a link where we can see the error occuring live,
